net-proxy/trojan/Manifest

```diff
@@ -0,0 +1,2 @@
+DIST trojan-1.7.0.tar.gz 36297 BLAKE2B 08096db9d25455bda5ee444ec946a9a0edd2f0ad080f451e19825df8eaf160111af168c5fc0cc5d185dc7b64c023c61356a10b2d81766853cd1503eacc512cf1 SHA512 d3ae67c06d07f10177907fa92a00765bc5e5221f888ff201175904e5a34733c99e8b3c18683d87a43272855a7dec02272502839e385cd73fc32eda0584ace039
```

Comment:
Also just the newest version, unless there's a very good reason to add both. Otherwise people would just install the newer and the older will never even be tested ;-).

Reply:
Okay, thanks